examples/integrations/README.md

```diff
@@ -4,51 +4,39 @@ Integrations are created for an API using the `integration:create` command. The
 
 **Note:** SwaggerHub On-Premise users need v. 1.26 to use the `integration:create` command.
 
+* [Amazon API Gateway Integration](#amazon-api-gateway-integration)
 * [API Auto Mocking Integration](#api-auto-mocking-integration)
-* [GitHub Integration](#github-integration)
-* [GitHub Enterprise Integration](#github-enterprise-integration)
 * [Azure DevOps Server Integration](#azure-devops-server-integration)
 * [Azure DevOps Services Integration](#azure-devops-services-integration)
 * [Bitbucket Cloud Integration](#bitbucket-cloud-integration)
 * [Bitbucket Server Integration](#bitbucket-server-integration)
+* [GitHub Enterprise Integration](#github-enterprise-integration)
+* [GitHub Integration](#github-integration)
 * [GitLab Integration](#gitlab-integration)
 * [Webhook Integration](#webhook-integration)
 
+## Amazon API Gateway Integration
+* **name**: Display name of the integration. Must be unique among all integrations configured for the given API version.
+* **configType**: "AMAZON_API_GATEWAY" is used to create an Amazon API Gateway integration.
+* **region**: AWS region where the API will be published. Possible values include: "us-east-1", "us-east-2", "us-west-1", "us-west-2", "eu-west-1", "eu-west-2", "eu-west-3", "eu-central-1", "eu-north-1", "ap-east-1", "ap-south-1", "ap-southeast-1", "ap-southeast-2", "ap-northeast-1", "ap-northeast-2", "sa-east-1", "cn-north-1", "cn-northwest-1", "ca-central-1", "me-south-1".
```

Comment:
Should I change it from "us-east-1" to `us-east-1`

Reply:
Probably yes, for consistency with other string value examples in this doc.